odl/discr/lp_discr.py

```diff
@@ -228,22 +222,29 @@ def meshgrid(self):
         """All sampling points in the partition as a sparse meshgrid."""
         return self.partition.meshgrid
 
-    def points(self, order=None):
+    def points(self, order='C'):
         """All sampling points in the partition.
 
         Parameters
         ----------
-        order : {None, 'C', 'F'}
-            Axis ordering in the resulting point array. For the default
-            ``None``, `default_order` is used.
+        order : {'C', 'F'}
```

Comment:
Shouldn't None be default_order?

Reply:
No, that will destroy views. `default_order` only applies to new elements.